### PR TITLE
Content

### PR DESCRIPTION
**Pull request by denis-maslovskiy**
Create product cards, which are buttons, and when clicked, a transition to the page with the selected product will be carried out.
The name of the game, author, genre and price will be displayed on the card. Also, it will be a picture with the game.

> Still on development stage

### CODE REVIEW

File: client/src/pages/HomePage.js [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
+    let goods = [1,2,3,4,5,6,7,8,9,10];
+
     return (
-        <h1>Home Page</h1>
+        <main className='container'>
+            <div className='content'>
+
+                {
+                    goods.map((item, index) => {
+
```
Comment (DashaKavalenka) [inline client/src/pages/HomePage.js:14]: prettify please

File: client/src/pages/HomePage.js [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+                        return (
+                            <button className='content__card-btn' key={index}>
```
Comment (DashaKavalenka) [inline client/src/pages/HomePage.js:16]: Index as a key is an anti-pattern

File: client/src/styles/content.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        color: rgb(97, 96, 96);
+        font-family: sans-serif;
+        margin: 1% 1%;
+        border-radius: 24px;
+
+        &__picture {
+            width: 150px;
+            height: 180px;
+            margin: 10px;
+            background-color: salmon;
```
Comment (DashaKavalenka) [inline client/src/styles/content.scss:57]: Use color designations in the same code style (Hex / rgba / color name). Create a file and put all the colors there as variables also for @media

Comment (DashaKavalenka) [reply client/src/styles/content.scss:57]: `$base-color: #EF3E5C;`
```
@mixin phone {
  @media (max-width: #{$phone-width}) {
    @content;
  }
}
```

File: client/src/pages/HomePage.js [hunk 4]
```diff
@@ -1,33 +1,40 @@
...
-        <main className='container'>
-            <div className='content'>
+  return (
+    <main className="container">
+      <div className="content">
+        {games.map((game) => {
+          return (
+            <button className="content__card-btn" key={game.id}>
+              <div className="content__card">
+                <div className="content__card__picture"></div>
```
Comment (jULIAmEleshk) [inline client/src/pages/HomePage.js:24]: You can not set element into element if you use BEM `content__card__picture`
In this case you can have block `contenet` and element `content__card`
And then `content__card` - it is block too. So  block - ` <div className="content__card card">` and then element
` <div className="card__picture"></div>`

File: client/src/pages/SelectedGame.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+      ? (position = Math.max(position, -2240))
+      : (position = Math.max(position, -1400));
+    ul.current.style.marginLeft = position + "px";
+    console.log(position);
+  };
+
+  return (
+    <div className="carousel">
+      <button
+        className="carousel__arrow carousel__arrow__prev"
```
Comment (jULIAmEleshk) [inline client/src/pages/SelectedGame.js:33]: The same for this className you can not use element into element. You have too use element as block
NOT:  `className="carousel__arrow carousel__arrow__prev"`

carousel__arrow - element of carousel
arrow - block
`__prev` - is this element or modificator?

File: client/src/styles/carousel.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    &__next {
+      border-radius: 0 15px 15px 0;
+    }
+  }
+
+  &__gallery {
+    margin: 0 auto;
+    width: 100%;
+    overflow: hidden;
+    // border: 2px solid $light-green-color;
```
Comment (jULIAmEleshk) [inline client/src/styles/carousel.scss:43]: remove useless code

File: client/src/styles/selected-game.scss [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+            margin-bottom: 5%;
+            @media (max-width: 382px) {
+                border-bottom: 2px solid $light-green-color;
+            }
+        }
+
+        // &__digital-copy {
+            
+        // }
+
```
Comment (jULIAmEleshk) [inline client/src/styles/selected-game.scss:47]: remove useless code

**Pull request merged**